src/tract.c

```diff
@@ -323,7 +323,7 @@ int is_read (ITEMSET *iset, FILE *file)
 
 /*--------------------------------------------------------------------*/
 
-int is_recode (ITEMSET *iset, int minfrq, int dir, int *map)
+int is_recode (ITEMSET *iset, int minfrq, int dir, int *map, int mode, int fullS))
```

Comment:
/*Makh2018: Adding mode and fullS as is_recode parameters to delete full support items in case of generator itemset mining*/